clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkClient.java

```diff
@@ -228,7 +229,8 @@ private void poll(long timeout, long now, boolean executeDelayedTasks) {
         trySend(now);
 
         // fail all requests that couldn't be sent
```

Comment:
The comment should either be removed, moved inside the `if` or updated. Personally, it doesn't seem to add much over the actual code, so I would remove it. But let's see what @hachikuji says.